_content/articles/the-sal-an-alternative-buddhist-holy-tree_ireland-John-D.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Sal — An Alternative Buddhist Holy Tree?"
+authors:
+  - "John D. Ireland"
```

Comment:
He's in the `_authors` folder: 
https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_authors/ireland.md

So for this line, just:

```yaml
  - ireland
```